public/js/controller/player-controller.js

```diff
@@ -95,7 +95,6 @@ export default class GameController {
         this.players = gameData.players;
         this.food = gameData.food;
         this.walls = gameData.walls;
-        this.gameView.showPlayerStats(gameData.playerStats);
```

Comment:
Denne trengs ikke lengmer, fordi den ble brukt  da vi hadde score list + notification box greiene. Nå prøver den å skrive til dommen for hver gang vi mottar ny state..